### DISCUSSION TITLE
Arch Linux AUR package available

### DISCUSSION THREAD

**Discussion by Blutkoete** [FyshOS/fynedesk#262] [Show and tell]
Hello,

thank you for your efforts. This is not really the same as official packaging, but I created an AUR package for FyneDesk.

https://aur.archlinux.org/packages/fynedesk

Works on my machines(tm)

**Comment by andydotxyz** [comment]
Oh that is fantastic, thanks for sharing :)
Feel free to play with the develop branch and see if anything warrants a new release to share.

**Reply by Blutkoete** [reply]
Good point, I added an AUR git package tracking the develop branch so I can compare.

https://aur.archlinux.org/packages/fynedesk-git

**Reply by andydotxyz** [reply]
Amazing thanks so much. Maybe you'd be interested in the login system too? (possibly needs more testing on different configurations) https://github.com/fyshos/fin

**Reply by Blutkoete** [reply]
https://aur.archlinux.org/packages/fin-git

I'll just do it slowly, but steadily :)

**Reply by andydotxyz** [reply]
Amazing thanks for that. Getting feedback from different setups etc is helpful as there are so many "corner cases" for this type of code!

**Comment by andydotxyz** [comment]
Maybe we could make some sort of packaging list on the README or elsewhere so people can follow the links and gain confidence :)
